docker-prompt

```diff
@@ -0,0 +1,22 @@
+
+#!/bin/sh
+case "$DOCKER_HOST" in
```

Comment:
the requirement is for play-with-docker to run `bash` in the dind containers instead of `ash`